bodhi/util.py

```diff
@@ -435,12 +435,10 @@ def request2html(context, request):
 def update2html(context, update):
     request = context.get('request')
 
-    if hasattr(update, 'title'):
-        title = update.title
-    else:
-        title = update['title']
```

Comment:
Yeah, confirmed.  This will work on some templates but not others.. The front page crashes with:

```python
MakoRenderingException: 

Traceback (most recent call last):
  File "/home/threebean/.virtualenvs/bodhi-python2.7/lib/python2.7/site-packages/pyramid_mako-1.0.2-py2.7.egg/pyramid_mako/__init__.py", line 148, in __call__
    result = template.render_unicode(**system)
  File "/home/threebean/.virtualenvs/bodhi-python2.7/lib/python2.7/site-packages/Mako-1.0.0-py2.7.egg/mako/template.py", line 452, in render_unicode
    as_unicode=True)
  File "/home/threebean/.virtualenvs/bodhi-python2.7/lib/python2.7/site-packages/Mako-1.0.0-py2.7.egg/mako/runtime.py", line 803, in _render
    **_kwargs_for_callable(callable_, data))
  File "/home/threebean/.virtualenvs/bodhi-python2.7/lib/python2.7/site-packages/Mako-1.0.0-py2.7.egg/mako/runtime.py", line 835, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/home/threebean/.virtualenvs/bodhi-python2.7/lib/python2.7/site-packages/Mako-1.0.0-py2.7.egg/mako/runtime.py", line 860, in _exec_template
    callable_(context, *args, **kwargs)
  File "/home/threebean/devel/bodhi/bodhi/templates/master.html", line 162, in render_body
    ${self.body()}
  File "/home/threebean/devel/bodhi/bodhi/templates/home.html", line 62, in render_body
    ${self.tables.updates(critpath_updates)}
  File "/home/threebean/devel/bodhi/bodhi/templates/tables.html", line 26, in render_updates
    ${fragments.update(update, display_user, display_request, display_release, display_karma)}
  File "/home/threebean/devel/bodhi/bodhi/templates/fragments.html", line 77, in render_update
    <td>${util.update2html(update) | n}</td>
  File "/home/threebean/devel/bodhi/bodhi/util.py", line 438, in update2html
    title = update.title
AttributeError: 'dict' object has no attribute 'title'
```